source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,135 @@
+
+#### Triggers and Actions
+ - Channel admins can now configure certain actions to be executed automatically based on trigger conditions without writing any code.
+
+#### Actions Restructure
+ - The **Actions** menu was restructured to reduce the clutter from Plugins and Apps.
+
+#### Playbook Updates
+ - Added retrospective metrics to track up to four key metrics that indicate performance of every run.
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged Calls v0.4.8 with Mattermost server v6.6.
+ - Added nested previews for permalinks.
+ - Added a right-hand side Channel Info panel to see and interact with channel information.
+ - Added support for inline editing of posts.
+ - Changed the Mattermost indigo theme to match the dark theme in code blocks.
+ - Updated in-product links from legacy domain about.mattermost.com to mattermost.com.
+ - Made it easier to copy code blocks by adding a copy button on hover.
+ - Made it easier to copy a message via a new **Copy Text** post menu item.
+ - Added a loading indicator to the **Threads** global list each time more posts are fetched on infinite scroll.
+ - Added search guidance to the **Threads** global list when no more posts can be loaded. This is only shown if you’ve scrolled to load older posts and reach the end of the list.
+ - Added accessibility support for custom statuses.
+ - Tooltip is now only displayed when text is too long in the announcement banner.
+ - When restricting direct messages to users on the same team, bots are now excluded from that restriction.
+ - Brazilian Portuguese language support was downgraded to Beta.
+ 
+#### Performance
+ - Improved performance when clearing notifications with Collapsed Reply Threads enabled.
+ - Improved performance of Collapsed Reply Threads when ``ThreadAutoFollow`` is enabled but ``CollapsedThreads`` is disabled.
+ - Fixed a potential memory leak in the sidebar when using accessibility hotkeys.
+ - Virtualized the emoji picker and added other performance improvements to the emoji picker.
+ - Improved the performance of storing users in webapp.
+ - Fixed a small memory leak in the **System Console**.
+
+#### Plugins
+ - Updated the plugin registry's ``registerCallButtonAction`` method to allow for displaying custom calls buttons in the channel header.
+ - Added a debugging setting to turn off client-side plugins for the current user.
+ - Added performance metrics related to plugin loading on page load.
+
+#### Administration
+ - The default for [``ThreadAutoFollow``](https://docs.mattermost.com/configure/configuration-settings.html#automatically-follow-threads) has been changed to ``true``. This does not affect existing configurations where this value is already set to ``false``.
```

Comment:
@esethna should we advise people to turn it on if they have it to off?

Reply:
Good point, I'll add a note